.env.example

```diff
@@ -159,6 +159,7 @@ CUSTOM_TOKEN_SECRET="ssosecret"
 ########################################################################
 # SSO provider configurations
 ########################################################################
+#SSO_PROVIDERS=google,facebook,instagram,linkedin
```

Comment:
`SSO_xxx_ENABLED` can be removed, because when it presents in `SSO_PROVIDERS`, its enabled.

Reply:
Updated. Thanks.